build-tools/packages/build-tools/src/type-test-generator/main.ts

```diff
@@ -12,12 +12,100 @@ import { PackageJson } from "../common/npmPackage";
 import { buildTestCase, TestCaseTypeData } from "../typeValidator/testGeneration";
 import { getFullTypeName, getNodeTypeData, TypeData } from "../typeValidator/typeData";
 import { typeOnly } from "./compatibility";
+import { ExtractorConfig } from "@microsoft/api-extractor";
 
 // Do not check that file exists before opening:
 // Doing so is a time of use vs time of check issue so opening the file could fail anyway.
 // Do not catch error from opening file since the default behavior is fine (exits process with error showing useful message)
 const packageObject: PackageJson = readJsonSync("package.json");
+const previousPackageName = `${packageObject.name}-previous`;
+
+const previousBasePath = path.join("node_modules", previousPackageName);
+
+const previousPackageJsonPath = `${previousBasePath}/package.json`;
+
+if (!existsSync(previousPackageJsonPath)) {
+	throw new Error(
+		`${previousPackageJsonPath} not found. You may need to install the package via pnpm install. Note that type tests logic looks specifically for a package named '${previousPackageName}'`,
+	);
+}
+
+/**
+ * Retrieves the alpha trimmed file path for type definitions from the API Extractor configuration.
+ * @returns {string} The path to the alpha trimmed type definitions file.
+ * @throws {Error} If api-extractor config cannot be loaded or if the alpha trimmed file path is undefined
+ */
+function getAlphaTypeRollupPathsFromExtractorConfig(): string {
```

Comment:
Nit: this is just returning a single path, so we should probably name this something like `getAlphaTypeRollupPathFromExtractorConfig`.

Reply:
Consider a more generic function that takes the roll up type as an argument. You can type it to only allow "alpha" if you must.